### PR TITLE
fix 605

### PR DESCRIPTION
**Pull request by williballenthin**
### Checklist

- [ ] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: capa/main.py [hunk 1]
```diff
@@ -777,8 +779,8 @@ def main(argv=None):
         )
         if args.tag:
             rules = rules.filter_rules_by_meta(args.tag)
+            logger.debug("selected %d rules", len(rules))
             for i, r in enumerate(rules.rules, 1):
-                logger.debug("selected %d rules", len(rules))
```
Comment (williballenthin) [inline capa/main.py:781]: unrelated fix

**Review by mr-tz — APPROVED** [review]
oh, great, thanks!
